depends/patches/qt/fix_qpainter_non_determinism.patch

```diff
@@ -0,0 +1,49 @@
+commit c12e7cfb36b1578f37f478681533f041fb0617df
+Author: Andrew Chow <achow101-github@achow101.com>
+Date:   Sat Nov 21 01:11:04 2020 -0500
+
+    Fix non-determinism?
```

Comment:
Can you add the relevant explanations to your patch file. Feel free to take any wording from [mine](https://github.com/bitcoin/bitcoin/pull/20440/commits/9276fe0216b05ec1bb2f68f6fa5fd517ba192cdb#diff-4f7c9172d8df0551d78b82b36f153091fbf71d742bab5431b7d8a16f068a1380). Also mention when the patch can be dropped.

Reply:
The patch file could be applied for the 0.21 branch only, no?

Reply:
Done.